cql2pgjson-cli/src/main/java/org/z3950/zing/cql/cql2pgjsoncli/CQL2PGCLIMain.java

```diff
@@ -0,0 +1,149 @@
+      ParseException {
+    Options options = new Options();
+
+    Option database = Option.builder("t")
+        .hasArg()
+        .required(true)
+        .desc("Postgres table name")
+        .build();
+
+    Option field = Option.builder("f")
+        .hasArg()
+        .required(false)
+        .desc("Postgres field name")
+        .build();
+
+    Option schema = Option.builder("s")
+        .hasArg()
+        .desc("Path to JSON schema file")
+        .build();
+
+    Option schemamap = Option.builder("m")
+        .hasArg()
+        .desc("A JSON string or a pathname to a JSON file to describe the db/schema map")
+        .build();
+
+    Option dbschema = Option.builder("b")
+        .hasArg()
+        .desc("Path to RMB-style schema.json to describe database")
+        .build();
+
+    options.addOption(schema);
+    options.addOption(database);
+    options.addOption(field);
+    options.addOption(schemamap);
+    options.addOption(dbschema);
+
+    CommandLineParser parser = new DefaultParser();
+    CommandLine line = parser.parse(options, args);
+    CQL2PgJSON cql2pgJson = null;
+    String fullFieldName = line.getOptionValue("t") + "." + line.getOptionValue("f", "jsonb");
+    if(!line.hasOption("m")) {
+      if(line.hasOption("b")) {
+        cql2pgJson = new CQL2PgJSON(fullFieldName, null, line.getOptionValue("b"));
+      } else if(line.hasOption("s")) {       
+        String schemaText = readFile(line.getOptionValue("s"), StandardCharsets.UTF_8);
+        cql2pgJson = new CQL2PgJSON(fullFieldName, schemaText);
+      } else {
+        cql2pgJson = new CQL2PgJSON(fullFieldName); //No schemas
+      }
+    } else {
+      Map<String, String> fieldSchemaMap = parseDatabaseSchemaString(line.getOptionValue("m"));
+      cql2pgJson = new CQL2PgJSON(fieldSchemaMap);
+    }
+    List<String> cliArgs = line.getArgList();
+    String cql = cliArgs.get(0);
+    return parseCQL(cql2pgJson, line.getOptionValue("t"), cql);
+  }
+
+  static String readFile(String path, Charset encoding) throws IOException
+  {
+    System.out.println("Reading file " + path);
+    byte[] encoded = Files.readAllBytes(Paths.get(path));
+    String content = new String(encoded, encoding);
+    //System.out.println(String.format("Content of %s is: %s", path, content));
+    return content;
+  }
+
+  static protected String parseCQL(CQL2PgJSON cql2pgJson, String dbName, String cql) throws IOException,
+      FieldException, SchemaException, QueryValidationException {
+    SqlSelect sql = cql2pgJson.toSql(cql);
+    String orderby = sql.getOrderBy();
+    logger.log(Level.FINE, String.format("orderby for cql query '%s' is '%s'", cql, orderby));
+    if(orderby != null && orderby.trim().length() > 0) {
+      return String.format("select * from %s where %s order by %s",
+          dbName, sql.getWhere(), orderby);
+    } else {
+       return String.format("select * from %s where %s",
+          dbName, sql.getWhere());
+    }
+  }
```

Comment:
```
    if (StringUtils.isBlank(orderBy)) {
      return String.format("select * from %s where %s",
            dbName, sql.getWhere())
    }
    return String.format("select * from %s where %s order by %s",
          dbName, sql.getWhere(), orderby);
```
What do you think about this?
* Using StringUtils.isBlank from org.apache.commons.lang3 (already used by cql2pg)
* Using a guard clause for the special case, and having the general case without nesting in an else clause.